File: 프로그래머스/2/12945. 피보나치 수/피보나치 수.py
def solution(n):
    answer = 0
    fibo = [0 for i in range(n+1)]
    fibo[0] = 0
    fibo[1] = 1
    for i in range(2,n+1):
        fibo[i] = fibo[i-1] + fibo[i-2]
    answer = fibo[-1] % 1234567
    return answer

File: 프로그래머스/2/12945. 피보나치 수/피보나치 수.js
function solution(n) {
    
    let arr = [0,1];
    for(let i =2; i<=n;i++){
        arr[i] = (arr[i-1] + arr[i-2]) % 1234567;
    }
    return arr[arr.length-1] % 1234567;
}
